kroxylicious-kms-provider-aws-kms-test-support/src/main/java/io/kroxylicious/kms/provider/aws/kms/AbstractAwsKmsTestKmsFacade.java

```diff
@@ -99,201 +55,6 @@ public final void stop() {
 
     @Override
     public final TestKekManager getTestKekManager() {
-        return new AwsKmsTestKekManager();
-    }
-
-    class AwsKmsTestKekManager implements TestKekManager {
```

Comment:
I'm not understanding why you've moved AwsKmsTestKekManager to the top level.

Reply:
The first approach for this PR was to start moving things out to refactor the Facades. During this process I make the changes to avoid duplicated code and remove the KEK existence checking.

I agree that it can difficult the review, so I will move again in the KekManagers and do that later in a future PR to refactor the facades.